src/include/uapi/drm_local/amdxdna_accel.h

```diff
@@ -413,6 +413,7 @@ struct amdxdna_drm_query_sensor {
 /**
  * struct amdxdna_drm_query_ctx - The data for single context.
  * @context_id: The ID for this context.
+ * @user_id: Firmware context ID.
```

Comment:
In driver, there are context and hardware context. I suggest we use hwctx_id.
```suggestion
 * @hwctx_id: Hardware context ID.
```